### PR TITLE
WIP Added fixers to remove redundant annotations

### PR DESCRIPTION
**Pull request by PetrHeinz**
| Q             | A
| ------------- | ---
|Description, reason for the PR| redundant info in annotations (takes space and requires maintenance)
|New feature| No 
|BC breaks| No 
|Fixes issues| #200 
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: packages/framework/src/Command/Exception/MissingLocaleException.php [hunk 1]
```diff
@@ -25,9 +25,6 @@ public function __construct($missingLocale, Throwable $previous = null)
         parent::__construct($message, 0, $previous);
     }
 
-    /**
-     * @return string
```
Comment (PetrHeinz) [inline packages/framework/src/Command/Exception/MissingLocaleException.php]: This should not have been removed!

File: packages/framework/src/Component/Grid/GridView.php [hunk 2]
```diff
@@ -118,9 +112,7 @@ public function renderBlock($name, array $parameters = [], $echo = true)
     }
 
     /**
-     * @param \Shopsys\FrameworkBundle\Component\Grid\Column $column
      * @param array|null $row
```
Comment (PetrHeinz) [inline packages/framework/src/Component/Grid/GridView.php]: This should have been removed as well (as the default value is clearly `null`). Seems like an issue in the fixers themselves, it shouldn't influence us as it isn't something critical...

File: packages/framework/src/Component/Grid/InlineEdit/AbstractGridInlineEdit.php [hunk 3]
```diff
@@ -57,9 +56,6 @@ public function getGrid()
         return $grid;
     }
 
-    /**
-     * @return bool
```
Comment (PetrHeinz) [inline packages/framework/src/Component/Grid/InlineEdit/AbstractGridInlineEdit.php]: This should not have been removed.

**Comment by TomasVotruba** [comment]
I think we had the same idea :)
There is more inspiration #390

File: packages/framework/src/Component/Grid/GridView.php (hunk 2 above)
Comment (TomasVotruba) [reply packages/framework/src/Component/Grid/GridView.php]: It' not the same: `?array` != `array`

File: packages/framework/src/Command/Exception/MissingLocaleException.php (hunk 1 above)
Comment (TomasVotruba) [reply packages/framework/src/Command/Exception/MissingLocaleException.php]: Please report these issues at Symplify so they can be fixed

File: packages/framework/src/Component/Grid/GridView.php (hunk 2 above)
Comment (PetrHeinz) [reply packages/framework/src/Component/Grid/GridView.php]: But argument declared as `array $var = null` is of type `array|null`, not `array`.

Comment (TomasVotruba) [reply packages/framework/src/Component/Grid/GridView.php]: I see. In that case some NullableFixer/Sniff might handle it

Comment (PetrHeinz) [reply packages/framework/src/Component/Grid/GridView.php]: You mean `NullableTypeForNullDefaultValueSniff` from Slevomat? That would change the typehint to `?array $var = null`, I think, and that's not intended here.

Comment (TomasVotruba) [reply packages/framework/src/Component/Grid/GridView.php]: Probably. Personally I prefer `?array $var` without default value, due to clean code

**Comment by Miroslav-Stopka** [comment]
Hi,
we discussed it with several developers, and we decided to annotate everything. This is the most standard way, and at the same time makes the annotation clear and consistent. Problem will be solved in #417. I'm closing this issue.

**Pull request closed without merging**